radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java

```diff
@@ -222,7 +201,7 @@ public Hash getHash() {
 	}
 
 	public AID getAid() {
-		return AID.from(getHash(), getShards());
+		return AID.from(getHash().toByteArray());
```

Comment:
Like mentioned in other library, this change I think would require some synchronization with the javascript library